adafruit_ssd1327.py

```diff
@@ -51,11 +51,25 @@
     b"\xAF\x00"  # DISPLAY_ON
 )
 
+
 # pylint: disable=too-few-public-methods
 class SSD1327(displayio.Display):
-    """SSD1327 driver"""
+    r"""SSD1327 driver
+
+    :param ~displayio.I2CDisplay bus: The data bus the display is on
+    :param \**kwargs:
+        See below
+
+    :Keyword Arguments:
+        * *width* (``int``) --
+          Display width
+        * *height* (``int``) --
+          Display height
+        * *rotation* (``int``) --
+          Display rotation
+    """
```

Comment:
This should be doable as such:

```suggestion
    """SSD1327 driver

    :param ~displayio.I2CDriver bus: The data bus the display uses
    :param int height: (keyword-only) The height of the screen
    :param int width: (keyword-only) The width of the screen
    :param int rotation: (keyword-only) The rotation.orientation of the
        screen, in degrees
    """
```

Reply:
The `(keyword-only` is just text in the description, but it's enough to get the point across